tests/console/start_systemd_testkit_offline.pm

```diff
@@ -0,0 +1,109 @@
+# SUSE's openQA tests
+#
+# Copyright 2020 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: install and execute the external systemd and installation must
+# run in offline mode
+# Maintainer: QE Core <qe-core@suse.de>
+
+use base 'consoletest';
+use testapi;
+use utils;
+use Mojo::JSON qw(encode_json);
+use strict;
+use warnings;
+
+my $log = '/tmp/systemd_run.log';
+my $testdir = '/usr/lib/systemd/test/SAP';
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+    assert_script_run("cd $testdir");
+    assert_script_run("tar -zxvf systemd_suse.tgz");
+    # Make the Network offline if OFFLIE_SUT is set to 1
+    # To be common criteria compliant, binaries not controlled by SUSE must be excecuted isolated
+    # hence the reason to disable the network here, beyond this step, treat the machine as tainted.
+    assert_script_run(qq{(ping -c4 build.suse.de && exit 1 || exit 0 )});
+    assert_script_run(qq{(ping -c4 8.8.8.8 && exit 1 || exit 0)});
+    record_info("START", "Testsuite execution is starting");
+    assert_script_run "cp systemd/*.sh .";
+    assert_script_run "ls -l .";
+    assert_script_run 'sh -e ./systemd_prepare.sh';
+    # Wait at least 180s for test to finish
+    my $wait = 180;
+    # Run the test and save the logs and results
+    # systemd_run.sh will fail with a non-zero retval if any of the sub-tests
+    # fail. We ignore it to parse the individual results from the log
+    my $out = script_output "su - abcadm -c '$testdir/systemd_run.sh' 2>&1 | tee $log", $wait, proceed_on_failure => 0;
+    record_info("END", "Testsuite excecution finished");
+    record_info("TEST LOG", "$out");
+    assert_script_run(qq{(ping -c4 build.suse.de && exit 1 || exit 0 )});
+    assert_script_run(qq{(ping -c4 8.8.8.8 && exit 1 || exit 0)});
+    $self->parse_results_from_output($out);
+    $self->upload_systemdlib_tests_logs;
+}
+
```

Comment:
I think that it would be useful to add a comment here on what this function `parse_results_from_output`